.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -1,71 +1,55 @@
-const fs = require('fs');
 const core = require('@actions/core');
 const github = require('@actions/github');
-
-// Use Github Actions' default environment variables to get repo information
-// https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
-const [repoOwner, repoName] = process.env.GITHUB_REPOSITORY.split('/');
+const functions = require('./functions');
 
 const MAX_RETRIES = 10;
-let errCount = 0;
-let shouldRetry;
-
-do {
-    shouldRetry = false;
-    // eslint-disable-next-line no-loop-func
-    exec('npm version prerelease -m "Update version to %s"', (err, stdout, stderr) => {
-        console.log(stdout);
-        if (err) {
-            console.log(stderr);
 
-            // It's possible that two PRs were merged in rapid succession.
-            // In this case, both PRs will attempt to update to the same npm version.
-            // This will cause the deploy to fail with an exit code 128
-            // saying the git tag for that version already exists.
-            if (errCount < MAX_RETRIES) {
-                console.log(
-                    'Err: npm version conflict, attempting to automatically resolve',
-                    `retryCount: ${++errCount}`,
-                );
-                shouldRetry = true;
-                const {version} = JSON.parse(fs.readFileSync('./package.json'));
-                const currentPatchVersion = `v${version.slice(0, -4)}`;
-                console.log('Current patch version:', currentPatchVersion);
+const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+let semanticVersionLevel = core.getInput('SEMVER_LEVEL', {require: true});
 
-                // Get the highest build version git tag from the repo
-                console.log('Fetching tags from github...');
-                const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
-                octokit.repos.listTags({
-                    owner: repoOwner,
-                    repo: repoName,
-                })
-                    .then((response) => {
-                        const tags = response.data.map(tag => tag.name);
-                        console.log('Tags: ', tags);
-                        const highestBuildNumber = Math.max(
-                            ...(tags
-                                .filter(tag => tag.startsWith(currentPatchVersion))
-                                .map(tag => tag.split('-')[1])
-                            ),
-                        );
-                        console.log('Highest build number from current patch version:', highestBuildNumber);
+// SEMVER_LEVEL defaults to BUILD
+// it actually would fall to build anyway, but I think it is better to make it explicitly
+if (!semanticVersionLevel || !Object.values(functions.semanticVersionLevels).find(v => v === semanticVersionLevel)) {
+    // eslint-disable-next-line max-len, semi
```

Comment:
I'd prefer if you didn't suppress eslint in this case, and only do it as a last resort.

Reply:
Sorry about that, I made the changes in another version and it ended up getting lost. I will move it here.

Reply:
I didn't want o break the line when writing and then forgot to remove the eslint-disable. Sorry about that!